src/accelerate/state.py

```diff
@@ -19,6 +19,14 @@
 import torch
 
 
+try:
+    import torch_ccl as ccl
```

Comment:
```suggestion
    import torch_ccl
```
I think this should take care of the problem